### PR TITLE
Rems 303 reinstantiate mongo

### PR DESCRIPTION
**Pull request by kennyEung**
## Describe your changes

Created two databases named `pims` and `remsadmin`. Also added a `mongo-init` script to seed the database with the aforementioned database and collections.

## Issue ticket number and Jira link

[JIRA](https://jira.mitre.org/browse/REMS-303)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: mongo-init.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+const dbPims = db.getSiblingDB('pims');
+const dbRemsAdmin = db.getSiblingDB('remsadmin');
+
+// Create Collections
+dbPims.createCollection('pimscollection');
+dbRemsAdmin.createCollection('remsadmincollection');
+
+// Insert document to ensure db/collection is created
+dbPims.pimscollection.insertOne({name: 'Hello World!'})
+dbRemsAdmin.remsadmincollection.insertOne({name: 'Hello World Again!'})
```
Comment (plarocque4) [inline mongo-init.js:11]: Should we remove the document after the collection is created?

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -16,13 +16,10 @@ services:
       context: ../test-ehr
       dockerfile: Dockerfile.keycloak
 
-  # Create test ehr container
```
Comment (plarocque4) [inline docker-compose-dev.yml:19]: Why were all of the comments removed?

File: mongo-init.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Create Databases
+const dbPims = db.getSiblingDB('pims');
+const dbRemsAdmin = db.getSiblingDB('remsadmin');
+
+// Create Collections
+dbPims.createCollection('pimscollection');
+dbRemsAdmin.createCollection('remsadmincollection');
```
Comment (plarocque4) [inline mongo-init.js:7]: Does this need to be an exhaustive list of all of the collections? Or can collections be added later by the applications?

Comment (smalho01) [reply mongo-init.js:7]: collections can be added later dynamically through the Mongo/Mongoose API.  Creating these collections doesn't really serve much as we will be creating different ones for use within the context of the code

Comment (plarocque4) [reply mongo-init.js:7]: Should they be removed then?

Comment (smalho01) [reply mongo-init.js:7]: yeah I think so

Comment (smalho01) [reply mongo-init.js]: I removed the inserting of documents, but unless we create a collection or do something with the db it doesn't actually show up as a new data base within the mongo client after testing. I renamed the collections to be tmp collections which works for now and should be fine.

File: docker-compose-dev.yml (hunk 2 above)
Comment (smalho01) [reply docker-compose-dev.yml:19]: which comments?

**Comment by smalho01** [comment]
also added to the  readme a command for running the docker instance without using the dev environment for those who prefer to just run locally

File: docker-compose-dev.yml (hunk 2 above)
Comment (plarocque4) [reply docker-compose-dev.yml:19]: The diff shows comments that were there like:
"# Create test her container"
"# Name of our service"

Comment (smalho01) [reply docker-compose-dev.yml:19]: okay, I'll add those comments back in

**Review by KeeyanGhoreshi — APPROVED** [review]
Looks good, works with #51

**Pull request merged**